tests/js/session-tests.js

```diff
@@ -177,6 +177,32 @@ module.exports = {
             });
     },
 
+    async testRealmOpenWithDestructiveSchemaUpdate() {
+        if (!isNodeProcess) {
+            return;
+        }
+
+        const partition = Utils.genPartition();
+
+        await runOutOfProcess(__dirname + "/download-api-helper.js", appConfig.id, appConfig.url, partition, REALM_MODULE_PATH);
+
+        const app = new Realm.App(appConfig);
+        const user = await app.logIn(Realm.Credentials.anonymous());
+        const config = getSyncConfiguration(user, partition);
+
+        throw new Error('Steffen threw a new Error()');
```

Comment:
💣

Reply:
`throw new Error('Steffen threw a new Error()');`
Lets keep it more generic and meaningful?

Reply:
OOOPS, that was a test NOT supposed to go in. Thanks for catching that!
(giant facepalm, sorry guys)

Reply:
oh no worries. I do this all the time so I can find test pieces I need to remove before merging.

Reply:
Well the dumbest part is that it was to showcase something to @RedBeard0531 in a conversation on slack... and 2 mins later he finds this genius line.. here 🙄

(also, normally @steffenagger wouldn't reference @steffenagger in the 3rd person, it was just for the end output 😬)